golang/cmd/factoryinsight/v2/services/service-tree.go

```diff
@@ -345,10 +345,10 @@ func GetCustomTagsTree(
 		// ignore underscores at the beginning or end of the tag or if there are multiple underscores in a row
 		sanitizedSplitTag := removeEmpty(splitTag)
 		// remove unused capacity to save memory
-		sanitizedSplitTag = slices.Clip(sanitizedSplitTag)
+		sanitizedSplitTag = slices.Clip(sanitizedSplitTag) // WTF is this?
```

Comment:
Removes unused capacity from the slice, to save a little bit of memory. I thought it wouldn't hurt